src/i18n/index.js

```diff
@@ -4,6 +4,8 @@ import enUS from "./en-us"
 const languages = [
     { name: "English", code: "en-us", flag: "us" },
     { name: "Russian", code: "ru", flag: "ru" }
+	{ name: "Spanish", code: "es", flag: "es" }
```

Comment:
There needs to be a comma after `{ name: "Russian", code: "ru", flag: "ru" }`. Make sure it's aligned properly too